atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -207,7 +207,7 @@ def run(self):
             self.log.debug("annotations: %r", annotations)
             raise
 
-        labels = self.make_labels()
+        labels = self.make_labels(metadata)
```

Comment:
But what you have here complicates the store_metadata plugin. It has no reason to do this.

If you want to avoid duplication, how about making a util.py function to get a named label from the build JSON metadata and have both koji_promote and sendmail use it?